examples/by_feature/local_sgd.py

```diff
@@ -130,8 +130,6 @@ def training_function(config, args):
     accelerator = Accelerator(
         cpu=args.cpu, mixed_precision=args.mixed_precision, gradient_accumulation_steps=gradient_accumulation_steps
     )
-    if accelerator.distributed_type not in [DistributedType.NO, DistributedType.MULTI_CPU, DistributedType.MULTI_GPU]:
```

Comment:
Redundant gaurd statement because this code is called when initializing LocalSGD